src/content/platform-overview/cluster-management/cluster-api/architecture/index.md

```diff
@@ -32,12 +32,37 @@ By default, the machines are split into three different failure domains or zones
 There is a machine created as a bastion that helps us with the operations. It is the single entry point to the running infrastructure. This way all the cluster machines can live in a private network and expose the services running on them via explicit configuration.
 
 {{< tabs >}}
-{{< tab id="flags-openstack" title="OpenStack">}}
+{{< tab id="flags-openstack" title="OpenStack (CAPO)">}}
 
 The setup requires an external network configured in the project to allow the machines to pull images or route requests from containers to the Internet. On the other side, there is an internal network which interconnects all master and worker machines allowing the internal communication of all containers within the cluster. At the same time, it allocates the load balancers that are created dynamically as result of exposing a service in the cluster. In case the load balancer is external, a floating IP is allocated to enable the connection with external endpoints.
 
 {{< /tab >}}
-{{< tab id="flags-clouddirector" title="VMware Cloud Director">}}
+{{< tab id="flags-vsphere" title="vSphere (CAPV)">}}
+
+### Compatibility
+
+The setup supports vSphere 6.7 Update 3 and above to have the Cloud Native Storage (CNS) feature. 
+
+### Authentication
+
+Cluster API Provider vSphere (CAPV), along with the associated Cloud Provider interface (CPI) and Container Storage interface (CSI), authenticate against the VMware vSphere API using credentials stored in a secret. 
+
+### Networking
+
+vSphere has no concept of load balancer so we leverage `kuve-vip` to provide load balancing in ARP mode (layer-2) for the kubernetes API, as well as the `kube-vip` cloud provider (CPI) for services of type `Load Balancer`.
+
+By default kube-vip requires to set IP addresses manually. To solve this, we install [cluster-api-ipam-provider-in-cluster](https://github.com/giantswarm/cluster-api-ipam-provider-in-cluster-app) in the clusters to control IP allocation through the use of `IPAddressClaims`. This has the benefit of being compatible with any vSphere environment, regardless of the network architecture, and avoid IP duplicates. This means that we recommend reserving a layer 2 subnet per management cluster.
```

Comment:
What's a layer 2 subnet? Maybe layer 3 (IP) is meant here?

Reply:
Nope, ARP is layer 2 🙂 
https://kube-vip.io/#arp

Reply:
Ah, didn't know that it's called "subnet" for MAC addresses as well. Makes sense.